src/lib/template-result.ts

```diff
@@ -100,7 +110,17 @@ export class TemplateResult {
 
   getTemplateElement(): HTMLTemplateElement {
     const template = document.createElement('template');
-    template.innerHTML = this.getHTML();
+    let value = this.getHTML();
+    const trustedTypes = window.trustedTypes;
+    if (trustedTypes && !policy) {
+      policy = trustedTypes.createPolicy('lit-html', {createHTML: (s) => s});
```

Comment:
Is creating a policy expensive at all? Why not move this into the `policy` initializer and assume it'll be used very shortly. A few less bytes and one less conditional in a perf-sensitive path.

Reply:
Moved up and made policy a const, as a hint to the VM that the check for policy being defined can be removed, it's either always defined or always undefined.